src/SignalR/clients/ts/signalr/tests/HubConnection.test.ts

```diff
@@ -273,7 +274,7 @@ describe("HubConnection", () => {
             });
         });
 
-        it.only("start completes if connection closes and handshake not received yet", async () => {
+        it("start completes if connection closes and handshake not received yet", async () => {
```

Comment:
Does this mean that this was the only test in the "invoke" suite running previously? Is there a way to automatically detect `it.only` and `describe.only` are being used and prevent it?

Reply:
It was the only test running for the entire file :(

I don't know of a way to detect it. We could perhaps add a check to see how many tests passed and if the number is less than 10 we know something is wrong.